update_ccsmp.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Needs go on the path, probably:
+# export PATH=$PATH:/usr/local/go/bin
+# Blindly following the instructions at https://sol-jira.atlassian.net/wiki/spaces/CPD/pages/2868052115/Golang+Process+and+Tools#Updating-CCSMP
+# Worked once, so saving for posterity.
+
+LOAD=/home/public/RND/loads/ccsmp/trunk/100.0.0.924/
```

Comment:
This is a public repository this is not allows in the git commit or history please remove

Reply:
Honestly I do no think we should put this script in this project please remove.

Reply:
I agree, we shouldn't be exposing the internal directory structure of our dev servers. We can put the script in the Confluence page that described the update process.

Reply:
I deleted the script.